### DISCUSSION TITLE
Hope to achieve voice interaction

### DISCUSSION THREAD

**Discussion by prodcd** [langgenius/dify#4689] [Help]
Now it is possible to convert text to voice, and I hope it can be played automatically instead of manually clicking again. The parameters for converting text to speech using the Qwen model are not yet complete.
I also hope to achieve voice input, automatically converting speech into text information and sending it to the model.
Thank you to the R&D team for their efforts.

**Comment by remisharrock** [comment]
same here, thanks for this idea !
